include/deal.II/lac/arpack_solver.h

```diff
@@ -522,7 +564,7 @@ template <typename VectorType,
           typename MatrixType2,
           typename INVERSE>
 inline void
-ArpackSolver::solve(const MatrixType1 & /*system_matrix*/,
```

Comment:
you probably should mention what `INVERSE` is for different modes.

Reply:
done